plugins/flytekit-inference/flytekitplugins/inference/nim/serve.py

```diff
@@ -131,12 +134,12 @@ def setup_nim_pod_template(self):
                 None,
             )
             if local_peft_dir_env:
-                mount_path = local_peft_dir_env.value
+                peft_mount_path = local_peft_dir_env.value
             else:
                 raise ValueError("NIM_PEFT_SOURCE environment variable must be set.")
```

Comment:
can you explain why `NIM_PEFT_SOURCE` should be set in the comment?
I'm ok with this, but I'm worrying about users might be confused

Reply:
@Future-Outlier can you +1 again please?